### PR TITLE
Use a floor round for geo coordinate aggregation.

### PR DESCRIPTION
**Pull request by pmclanahan**
Install nose.

This is another aggregation experiment. This "should" get things
to converge even a litte more.

@jgmize r?

### CODE REVIEW

**Comment by jgmize** [comment]
```
$ ./manage.py test smithers
Creating test database for alias 'default'...
E
======================================================================
ERROR: tests.test_lisa (unittest.loader.ModuleImportFailure)
----------------------------------------------------------------------
ImportError: Failed to import test module: tests.test_lisa
Traceback (most recent call last):
  File "/usr/lib/python2.7/unittest/loader.py", line 252, in _find_tests
    module = self._get_module_from_name(name)
  File "/usr/lib/python2.7/unittest/loader.py", line 230, in _get_module_from_name
    __import__(name)
  File "/home/jgmize/mrburns/smithers/tests/test_lisa.py", line 5, in <module>
    import lisa
  File "/home/jgmize/mrburns/smithers/lisa.py", line 31, in <module>
    parser.add_argument('--file', default=conf.GEOIP_DB_FILE,
AttributeError: 'module' object has no attribute 'GEOIP_DB_FILE'


----------------------------------------------------------------------
Ran 1 test in 0.001s

FAILED (errors=1)
```

**Comment by pmclanahan** [comment]
`cd smithers && nosetests`

**Comment by pmclanahan** [comment]
also you need to tell it where a db file is I guess... hmm

**Comment by pmclanahan** [comment]
no good default for that, though I guess just having it exist would make this pass.

**Comment by pmclanahan** [comment]
@jgmize also that setting is in the local.py-dist. should pass if you just do `cp smithers/smithers/conf/local.py{-dist,}`

File: smithers/lisa.py [hunk 1]
```diff
@@ -138,11 +140,18 @@ def main():
...
+                statsd.gauge('queue.geoip', redis.llen(rkeys.IPLOGS))
 
 
 if __name__ == '__main__':
+    try:
+        geo = maxminddb.Reader(args.file)
+    except IOError:
+        log.error('ERROR: Can\'t find MaxMind Database file (%s). '
+                  'Try setting the --file flag.' % args.file)
+        sys.exit(1)
```
Comment (jgmize) [inline smithers/lisa.py:156]: why put this here instead of in main() itself?

Comment (pmclanahan) [inline smithers/lisa.py:156]: `geo` is a global. I could put it in main and use the `global` keyword if you like that better.

**Pull request merged**